python/lambdas/copy-files-from-tdr/lambda_function_test.py

```diff
@@ -0,0 +1,167 @@
+import math
+import os
+import unittest
+from unittest.mock import patch
+import lambda_function
+
+
+@patch.dict('os.environ', {'DESTINATION_BUCKET': 'destination-bucket'})
+@patch.dict('os.environ', {'DESTINATION_QUEUE': 'destination-queue'})
+class TestLambdaFunction(unittest.TestCase):
+    @patch('lambda_function.s3_client.head_object')
+    @patch('lambda_function.s3_client.copy_object')
+    @patch('lambda_function.sqs_client.send_message')
+    def test_standard_copy(self, mock_send_message, mock_copy_object, mock_head_object):
+        mock_head_object.return_value = {'ContentLength': 1024}  # 1 KB
+        event = {
+            'Records': [
+                {'body': '{"bucket": "source-bucket","fileId":"test-file"}'}
+            ]
+        }
+        context = {}
+        lambda_function.lambda_handler(event, context)
+
+        def assert_copy_object_arguments(idx, key):
+            expected_response = \
```

Comment:
Shouldn't this be `expected_args` or `expected_request`?

Reply:
It's `expected_request` yeah.